test/common/fixtures/assetUpgrader.ts

```diff
@@ -1,24 +1,10 @@
-import {
-  deployments,
-  ethers,
-  getNamedAccounts,
-  getUnnamedAccounts,
-} from 'hardhat';
+import {ethers, getNamedAccounts, getUnnamedAccounts} from 'hardhat';
 import {BigNumber, Contract} from 'ethers';
 import {waitFor} from '../../utils';
 import {transferSand} from '../../polygon/catalyst/utils';
 
 // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
 export const assetUpgraderFixtures = async () => {
```

Comment:
Why no deployment.fixture in this common fixture ?
I thought every fixture should have a deployment

Reply:
Is not mandatory.
We will move all the calls to `deployment.fixture` to the `withSnapshot` call.
The idea is to have one call to `crateFixture` and one call to `fixture` max and in the right order (we do that inside `withSnapstho`)